commands/version/index.js

```diff
@@ -476,7 +476,7 @@ class VersionCommand extends Command {
   }
 
   updatePackageVersions() {
-    const { conventionalCommits, changelogPreset, changelog = true } = this.options;
+    const { conventionalCommits, changelogPreset, changelogHeader, changelog = true } = this.options;
```

Comment:
This option needs explicit definition in the [yargs options](https://github.com/lerna/lerna/blob/91d9d3e597db6578b3be7f8ee88d1246ad0f80c5/commands/version/command.js#L35) for the version command.

It also needs documentation in the command's [README](https://github.com/lerna/lerna/blob/master/commands/version/README.md). Since it mostly makes sense as a config in lerna.json, don't forget an example of that config path.